packages/lightning-lsp-common/src/resources/sfdx/typings/lds.d.ts

```diff
@@ -656,6 +719,57 @@ declare module 'lightning/analyticsWaveApi' {
         frequency: 'daily' | 'Daily';
     }
 
+    /**
+     * Wave Data Connector input representation
```

Comment:
Change Wave to Tableau CRM or just remove for "Data Connector input representation"